plugins/Coala/Coala.py

```diff
@@ -0,0 +1,16 @@
+import re
+
+from errbot import BotPlugin, re_botcmd
+
+class Coala(BotPlugin):
+
+    def callback_message(self, msg):
+        """
+        Shout at the user for using an upercased c in coala.
+        """
+
+        if "Coala" in msg.body:
+            self.send(msg.frm,
+                      "What, what, did someone say `Coala`? @{} you're on the blacklist, "
+                      "you'll tracked down until you write a bear for `coala`. "
+                      "And remember, `coala` is ALWAYS written in lowercase `c`.!!".format(str(msg.frm).split('@')[0]))
```

Comment:
The original bot doesn't work on `Coala` it works on a regex. That takes any spelling of coala, eg: CoaLa